mlos_core/mlos_core/optimizers/__init__.py

```diff
@@ -62,6 +62,7 @@ class OptimizerFactory:
     @staticmethod
     def create(*,
                parameter_space: ConfigSpace.ConfigurationSpace,
+               optimization_targets: List[str],
```

Comment:
Shouldn't this be:
```suggestion
               optimization_targets: List[Tuple[str, Literal[min, max]]],
```

Reply:
Ah, nm, this was for the mlos_core optimizer, which we always minimize?

Reply:
exactly. in mlos_core we always minimize